charts/gardener/provider-local/internal/shoot-system-components/charts/local-path-provisioner/templates/rolebinding.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if not .Values.pspDisabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: gardener.cloud:psp:local-path-provisioner
+  namespace: kube-system
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: gardener.cloud:psp:kube-system:local-path-provisioner
+subjects:
+- kind: ServiceAccount
+  name: local-path-provisioner-service-account
+  namespace: kube-system
+{{- end }}
```

Comment:
You added the resource to a `ClusterRole` for which the `ClusterRoleBinding` is already present.
You can either create a separate `Role` and keep this file or remove this file. I would suggest to keep the PSP resources seperate.